### DISCUSSION TITLE
Couldn't install this library at my Expo Project

### DISCUSSION THREAD

**Discussion by FooJiaJieng** [mmazzarolo/react-native-modal-datetime-picker#633] [Q&A] [answered]
<img width="1186" alt="Screenshot 2022-03-17 at 8 22 27 PM" src="https://user-images.githubusercontent.com/80948584/158807964-e92a5bdf-507c-46f5-a4c4-0c3e41f8e091.png">
<img width="1195" alt="Screenshot 2022-03-17 at 8 22 47 PM" src="https://user-images.githubusercontent.com/80948584/158807980-a603626b-125d-44b8-9e76-588b69fae8c3.png">

I tried the cmd line for expo provided, 'expo install react-native-modal-datetime-picker @react-native-community/datetimepicker'.
But still unable to install the libraries. 
My react native version is 0.64.3.
Do this library only support project with react native 0.65.0 and above?

**Comment by mmazzarolo** [comment, marked as answer]
@FooJiaJieng yup, see https://github.com/mmazzarolo/react-native-modal-datetime-picker/issues/627 -- PRs for updating the README are welcome!

**Reply by FooJiaJieng** [reply]
Thank you very much, I ran this and it works. "expo install react-native-modal-datetime-picker@10.2.0  @react-native-community/datetimepicker"
